### ISSUE TITLE
Fix documentation on enabling and disabling labextensions

### ISSUE THREAD

**Issue by ykazakov** [jupyter/nbgrader#1921]
In version 0.9.3 the location of labextensions has changed from `nbgrader` to `@jupyter/nbgrader`. #1893

Hence [enabling and disabling of labextensions](https://nbgrader.readthedocs.io/en/stable/user_guide/installation.html) should now be done with this prefix:

```
jupyter labextension disable @jupyter/nbgrader:formgrader
jupyter labextension disable @jupyter/nbgrader:assignment-list
jupyter labextension disable @jupyter/nbgrader:course-list
jupyter labextension disable @jupyter/nbgrader:create-assignment
jupyter labextension disable @jupyter/nbgrader:validate-assignment
```

...

```
jupyter labextension enable --level=user @jupyter/nbgrader
jupyter server extension enable --user --py nbgrader
```

...

```
jupyter labextension disable --level=sys_prefix @jupyter/nbgrader:assignment-list
jupyter server extension disable --sys-prefix nbgrader.server_extensions.assignment_list
```

**Comment by brichet** [comment]
Thanks @ykazakov for opening this, #1927 should fix it.

**Comment by brichet** [comment]
Fixed by #1927
